### PR TITLE
How We Use Next.js for Frontend Development

### PR DESCRIPTION
**Pull request by Snailed**

### CODE REVIEW

File: how-we-use-nextjs.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+See [MitPensum-frontend](https://github.com/Kvalifik/mitpensum-frontend/tree/staging/store) as an example of how to use Redux as a caching layer.
+
+## Communicating with backends
+We use Axios as a network request library, and pack all of our request handling into neat React Hooks that can be imported into any component. See [MitPensum-frontend](https://github.com/Kvalifik/mitpensum-frontend/tree/staging/hooks).
+
+## Styling
+We use TailwindCSS for styling. See [kiddo-frontend](https://github.com/Kvalifik/kiddo-frontend) for good example.
+
+## How to setup a project using Next.js
+To set up a Next.js project, it is best to duplicate an existing project. The [kiddo-frontend](https://github.com/Kvalifik/kiddo-frontend) is a great starting point.
```
Comment (widding) [inline how-we-use-nextjs.md:18]: Perhaps we should setup a template repository á la Shopify Skeleton Theme? This way we don't get people creating duplicate projects and forgetting to change the original name in different places.

Maybe even extend Kvalifik-CLI to bootstrap based on a template and populate the correct name?

Comment (nikolajvonholck) [reply how-we-use-nextjs.md:18]: We might do this in the future, but right now we don't really have time to maintain such a template repo.

**Review by nikolajvonholck — APPROVED** [review]
@Snailed see my minor changes. If you agree, then merge :)

**Comment by nikolajvonholck** [comment]
@Snailed

**Pull request merged**